src/scripts/components/NoticeMeta.js

```diff
@@ -1,15 +1,15 @@
-import * as moment from 'moment/moment';
+import { noticeDateFormat } from '../utils';
 
 /**
  * The notice metadata, for example the source of the notification or the date
  *
  * @param {Object} props
- * @param {number} props.date   The date of the notification.
+ * @param {string} props.date   The date of the notification.
```

Comment:
The date is a string?

Reply:
seems the [dateI18n accepts](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-date/#datei18n) only stings and not numbers